src/Serilog.Sinks.AzureTableStorage/Serilog.Sinks.AzureTableStorage.csproj

```diff
@@ -22,6 +22,7 @@
 
   <ItemGroup>
     <PackageReference Include="Serilog" Version="2.6.0" />
+    <PackageReference Include="Serilog.Formatting.Compact" Version="1.0.0" />
```

Comment:
If we keep using the old JSON formatter by default, we can drop this dependency.